cli/src/opts/cast.rs

```diff
@@ -298,6 +301,17 @@ pub enum Subcommands {
         #[structopt(subcommand)]
         command: WalletSubcommands,
     },
+    #[structopt(name = "interface", about = "generate contract's interface from ABI")]
+    Interface {
+        #[structopt(help = "The contract's address")]
+        contract_address: Option<String>,
+        #[structopt(help = "pragma version")]
+        pragma: Option<String>,
+        #[structopt(help = "chain")]
+        chain: Option<ethers::core::types::Chain>,
```

Comment:
You can use `#[structopt(default_value = "mainnet")]` and make `chain: Chain` instead of `chain: Option<Chain>`